Multisig/Cross-layer/Configuration/Config.Example.xcconfig

```diff
@@ -176,3 +176,10 @@ ASSETCATALOG_COMPILER_APPICON_NAME  = AppIcon_$(SERVICE_ENV)
 //
 // Also, Firebase Analytics requires clients to add -ObjC linker option.
 OTHER_LDFLAGS = $(inherited) -fprofile-instr-generate -ObjC
+
+// Set in https://console.cloud.google.com/apis/credentials?project=web3auth-safe&supportedpurview=project
```

Comment:
I would say we need to define all of those three as environment variables in Jenkins, at least the first two

Reply:
Added here: https://padmeamidala.ngrok.io/credentials/store/system/domain/_/